fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsJobManager.java

```diff
@@ -80,16 +80,24 @@ public Map<Long, StatisticsJob> getIdToStatisticsJob() {
     }
 
     public void createStatisticsJob(AnalyzeStmt analyzeStmt) throws UserException {
-        // step1: init statistics job by analyzeStmt
-        StatisticsJob statisticsJob = StatisticsJob.fromAnalyzeStmt(analyzeStmt);
-        writeLock();
```

Comment:
Seems like the read lock is never used, so why not use object lock directly

Reply:
sure